### DISCUSSION TITLE
Testing environments on demand

### DISCUSSION THREAD

**Discussion by xperjon** [abacritt/angularx-social-login#411] [General]
Hi!
We're currently using this project in our Angular web app and now we have plans to setup testing environments on demand. This will imply that every feature will get deployed to its own subdomain, e.g. feature1.company.org, for testing. 
Now to the issue - How to handle redirect url?
Since every new testing environment will have its own subdomain, how should we dynamically be able to configure the redirect url for that environment?
Does anyone know if this is possible with this library or if there is any other solution?

**Comment by jaibatrik** [comment]
This library does not really need any redirect URL as such. The library currently supports only client-side authentication providers.
